prelude/prelude.purs

```diff
@@ -547,6 +660,7 @@ module Prelude
 
   infixr 5 <>
 
+  -- | Concatenation-like operation.
```

Comment:
I don't think this adds anything. If anything, it would confuse those unfamiliar with it.